internal/wire/parse.go

```diff
@@ -214,6 +218,21 @@ type InjectorArgs struct {
 	Pos token.Pos
 }
 
+// Field describes a list of fields from a struct.
+type Field struct {
+	// Parent is the struct the field belongs to.
+	Parent types.Type
+	// Name is the field name.
+	Name string
+	// Pkg is the package that the struct resides in.
+	Pkg *types.Package
+	// Pos is the source position of the func keyword or type spec
```

Comment:
Comment stale, perhaps:

```suggestion
	// Pos is the source position of the field declaration.
```